### PR TITLE
Add read the docs

### LINKED ISSUE #19: Add readthedocs
When we got the first material on master I will look into generation docs with with sphinx and readthedocs.org. 
This way the project can be read like a book and even downloaded as pdf.
To generate docs from notebooks, [nbsphinx](https://github.com/spatialaudio/nbsphinx) will be used.

### PR DESCRIPTION
**Pull request by s-weigand**
This PR adds this project to read the docs, so users can read it online or download the material as ``ziped-html`` and ``epub`` (``pdf`` doesn't work atm due to problems with svg badges, might need to further look into [``npsphinx``](https://github.com/spatialaudio/nbsphinx)).
I had to change the folder and file names since the ``#`` in the paths did mess up, links to binder.

@redbluee Can please review ``docs/getting_started.rst``, since this is the only text i wrote, the rest is pure tooling.

closes #19

### CODE REVIEW

**Comment by s-weigand** [comment]
Now PDF is working 😄

File: docs/getting_started.rst [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+===============
+Getting started
+===============
+
+Run it locally
+--------------
+
+This is the ideal way, since you will have everything you need
+in you day to day student life installed on you computer and
```
Comment (redbluee) [inline docs/getting_started.rst]: **your** day to day student life installed on **your** computer

File: docs/getting_started.rst [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+all your data will persist.
+
+Install python
+^^^^^^^^^^^^^^
+
+Anaconda
+""""""""
+Anaconda is our recommended to install python on all operating systems,
```
Comment (redbluee) [inline docs/getting_started.rst]: Anaconda is our **recommendation**

File: docs/getting_started.rst [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+since it comes with most needed packages prebundled (batteries included).
+Another bonus is that ``conda`` isn't just a python package manager like
+``pip``, but a package manager for multiple resources
+(i.e. `node.js`_ or `latex`_) and also an `environment`_ manager.
+Yet another bonus of ``conda`` is that is has a build pipeline in place,
```
Comment (redbluee) [inline docs/getting_started.rst]: is that **it** has a build pipeline in place

File: docs/getting_started.rst [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+    If you are using a Posix system (Linux/OsX) you don't want to mess
+    with the system python, since many system tools rely on it,
+    and in a worse case scenario you could break your Os.
+
+Pure CPython
+""""""""""""
+
+
+If you don't want to install conda, this `Python installation guide`_ can guide
+you through the process tof getting the pure CPython Interpreter.
```
Comment (redbluee) [inline docs/getting_started.rst]: through the process ~~t~~of getting the pure CPython Interpreter

**Review by redbluee — CHANGES_REQUESTED** [review]
nice explanation

**Comment by s-weigand** [comment]
thanks 😊

**Review by redbluee — APPROVED** [review]

**Comment by s-weigand** [comment]
@all-contributors please add @redbluee for review

**Comment by s-weigand** [comment]
@all-contributors please add @redbluee for doc infra maintenance

**Comment by s-weigand** [comment]
@all-contributors please add @s-weigand for doc infra maintenance

**Comment by s-weigand** [comment]
@all-contributors please add @redbluee for ideas projectManagement maintenance

**Comment by s-weigand** [comment]
@all-contributors please add @s-weigand for doc infra maintenance ideas projectManagement

**Pull request merged**